api/aggregators/searchAggregator.js

```diff
@@ -108,10 +108,25 @@ exports.createMatchAggr = async (schemaName, projectId, keywords, caseSensitive,
  */
 exports.createSortingPagingAggr = function(schemaName, sortValues, sortField, sortDirection, pageNum, pageSize) {
   const searchResultAggregation = [];
-  let defaultTwoSorts = false;
+  let datePostedHandlingTruncating = false;
+  if (sortField && sortField.includes(",") || Object.keys(sortValues).length > 1){
+    //sort will have multiple values passed
+    if (sortField.includes("datePosted") || sortValues.hasOwnProperty("datePosted")){
+      //datePosted is too specfic(in it's time) and needs the truncated form of date, can be expanded if other dates are required to be truncated
+      tempSortValues = {};
+      for (let property in sortValues){
```

Comment:
Can `sortValues` be blank if `sortField` is an array? Or if `sortField` exists then should there also be `sortValues`?

Reply:
No, Yes.